src/app/home/sponsors/sponsors.component.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../../styles-mat-variables";
+
+.sponsor {
+  margin: 40px;
+}
+
+.sponsors-logo {
+  max-width: 80%;
+  max-height: 40%;
+
+  @media #{$medium-and-up} {
+    max-width: 100%;
+  }
+
+  .become-sponsor-button {
```

Comment:
As I told you, this should be an id instead of a class because you are using this only for one element:
#become-sponsor-button.